src/shipit_code_coverage/shipit_code_coverage/taskcluster.py

```diff
@@ -90,10 +90,24 @@ def is_coverage_task(task):
     return any(task['task']['metadata']['name'].startswith(t) for t in TEST_PLATFORMS)
 
 
-def get_suite_name(task):
+def get_chunk_name(task):
     name = task['task']['metadata']['name']
     for t in TEST_PLATFORMS:
         if name.startswith(t):
             name = name[len(t) + 1:]
             break
-    return '-'.join([p for p in name.split('-') if p != 'e10s' and not p.isdigit()])
+    return '-'.join([p for p in name.split('-') if p != 'e10s'])
+
+
+def get_suite_name(chunk_name):
+    return '-'.join([p for p in chunk_name.split('-') if not p.isdigit()])
```

Comment:
It looks like previous suite names could not contain `e10s`. Now they can?

Reply:
Now the parameter of `get_suite_name` will be a `chunk_name`, where `e10s` string should have already been removed.
This function is actually unused now, but I will change both `get_chunk_name` and `get_suite_name` in the near future to use the information from the task itself (https://github.com/mozilla-releng/services/issues/759), which should be less brittle than parsing the name of the task.